src/core/engines/Cesium/Feature/Polygon/index.tsx

```diff
@@ -17,13 +17,21 @@ import {
   type FeatureProps,
 } from "../utils";
 
-export type Props = FeatureProps<Property>;
+export type Props = FeatureProps<Property> & { disableWorkaround?: boolean };
```

Comment:
When do we use this option?

Reply:
It is mostly for debugging purposes as there is no opportunity to use it. When deleting a workaround someday, the implementation will make it easier to see where the workaround should be deleted.